src/Cli/dotnet/commands/dotnet-workload/elevate/LocalizableStrings.resx

```diff
@@ -118,9 +118,9 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="CommandDescription" xml:space="preserve">
-    <value>Start the elevate server process to facilitate MSI based installations</value>
+    <value>Start the elevate server process to facilitate MSI based installations.</value>
```

Comment:
```suggestion
    <value>Start the elevated server process to facilitate MSI based installations.</value>
```

Reply:
This command description probably isn't needed at all, since this command is hidden and won't show up in help.